metadeploy/consumers.py

```diff
@@ -109,9 +109,9 @@ def has_good_permissions(self, content):
 
     def handle_org_special_case(self, content):
         if content["model"] == "org":
-            content["id"] = convert_org_id_to_key(self.scope["user"].org_id)
+            content["id"] = convert_org_id_to_key(content["id"])
```

Comment:
@wlonk This is an okay change? Necessary to allow anonymous users to subscribe to org-changes.

Reply:
Yeah, given that it's just for orgs.